test-cases/longevity/longevity-large-partition-2days.yaml

```diff
@@ -12,7 +12,7 @@ prepare_write_cmd:  ["scylla-bench -workload=sequential -mode=write -replication
                      "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=100 -clustering-row-count=50000 -partition-offset=701 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150"
 ]
 stress_cmd: ["scylla-bench -workload=uniform -mode=read -select-order-by=desc -replication-factor=3 -partition-count=800 -clustering-row-count=50000 -clustering-row-size=uniform:100..5120 -rows-per-request=2000 -timeout=240s -retry-number=10 -concurrency=400 -duration=44h -connection-count 400",
-             "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=200 -clustering-row-count=50000 -partition-offset=801 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150"
+             "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=200 -clustering-row-count=50000 -partition-offset=601 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150"
```

Comment:
Not clear.
Please explain better in the commit message.
Don't assume we know what the validation_partition does and why we have offset.

Reply:
fixed:
```
fix(large-partitions-2d): Adjust s-b offset for validate_partitions
	the s-b offset of stress-cmd shouldn't exceed the
	offset of prepare-cmd for validate_partitions validation
	Validate-partitions probably counted rows for the 800 partitions,
	created in prepare step. then stress-cmd added 200 NEW partitions.
	Then, i guess, validate_partition found differences, most probably
	about these new 200 partitions.
	Fixes: https://github.com/scylladb/scylla-cluster-tests/issues/6183
```

Reply:
The fix is correct.
But I agree with @roydahan that the description is not clear.
I propose to use following commit description:

```
fix(large-partitions-2d): make validate-partitions work

We have the feature in the SCT where we count number of partitions
and number of rows there then save it to a dict.
Run regular stress load and after it's finish we gather
that partition-rows mapping info again.

In case of the "large-partitions-2d" we use 2 stress commands
where 1 of them is of "write" mode.
And the problem with it is that it creates 200 new partitions
in addition to the 800 created in the "prepare" step.
It inevitably leads to the diff of 2 mappings
where we have 800 and 1000 key-value pairs accordingly.

So, fix it by making the "write" stress load rewrite existing 200 partitions
not adding new ones.
In this case the "validate_partitions" feature won't be spoiled.
```

Reply:
@yarongilor , this PR may be linked to other source with many information, and these links can be found in the commit message, but the message above from @vponomaryov makes everyone understand the problem, and the solution.

not because a PR is linked to other source of information, we are ok to leave the commit message without better context... as we spoke many times in our QA forum meetings, there is no problem having long commit messages, if they are clear... and we can add links to issues and other PRs related, but it must be self contented in terms of information.

please update the commit message, so we can merge this PR